.circleci/config.yml

```diff
@@ -35,6 +35,10 @@ jobs:
           name: Validate pytest-benchmark invocation of training suite
           command: ./scripts/run_bench_and_upload.sh
           no_output_timeout: "60m"
+      - run:
+          name: Sanity check for lazy_bench.py training suite
+          command: python lazy_bench.py --just_run_once  --run_in_subprocess alexnet --test=eval
+          no_output_timeout: "60m"
```

Comment:
How long will this command normally take? Does it require GPU?

Reply:
it runs one inference of AlexNet on GPU which only takes a few seconds. I could switch it to running on CPU if  it's more cost effective for us.